### PR TITLE
Fixes for tutorials after regenesis + changelog

### PR DESCRIPTION
**Pull request by uF4No**
Fixed getting started on Hardhat page.
Fixed quickstart tutorial although I haven't been able to test paying transactions with the testnet paymaster.
Updates installation scripts throughout the whole docs to use ethers 5.7.x with zksync-web3 0.13.
Updates compiler version throughout the whole docs to 1.3.1
Includes warning message in untested tutorials.

Add changelog page in "Troubleshooting" section of the menu

### CODE REVIEW

**Review by krofax — DISMISSED** [review]

File: docs/dev/troubleshooting/changelog.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+This update included multiple changes in the system to prepare it for our “Fair Onboarding Alpha” milestone. These updates include:
+
+- a fee mechanism revamp that replaces `ergs` with `gas` to make it easier to understand (after all, we’re all part of the Ethereum ecosystem).
+- updates in the Account Abstraction support
+- changes in L1 and L2 system contract interfaces
+- breaking changes in our SDKs where methods and properties that contained `ergs` are renamed `gas`
+- and some API error messages now follow the Geth format (more updates coming soon).
+
+**These udpates were followed by a system regenesys.**
```
Comment (StanislavBreadless) [inline docs/dev/troubleshooting/changelog.md:13]:
```suggestion
**These updates were followed by a system regenesis.**
```

File: docs/dev/troubleshooting/changelog.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### How to update your project
+
+To update your project, follow these steps:
+
+- Update all your project dependencies, while continuing to use `ethers v5.7.x` (as `ethers v6.x` introduces additional breaking changes).
+- Delete the `artifacts-zk` and `cache-zk` folders.
+- If you’re using the local-setup to run unit tests, update the docker images with `docker-compose pull` and restart the local setup state by running the `./clear.sh` script.
```
Comment (StanislavBreadless) [inline docs/dev/troubleshooting/changelog.md:21]: @uF4No did we update + test the local node?

File: docs/dev/troubleshooting/changelog.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Re-compile your project with the latest version of the binary compiler (`v1.3.1`).
+- Update `zksync-web3` to `0.13.x`. All methods that included `ergs` have been renamed to `gas` and some function signatures have changed.
+- If your project uses zksync system contracts, make sure to update the `@matterlabs/zksync-contracts` package as well. There are changes in multiple contract interfaces.
```
Comment (StanislavBreadless) [inline docs/dev/troubleshooting/changelog.md:24]: cc @vladbochok please publish the package

File: docs/dev/troubleshooting/changelog.md (hunk 2 above)
Comment (uF4No) [reply docs/dev/troubleshooting/changelog.md:21]: I was told the docker images were going to be updated

File: docs/dev/troubleshooting/changelog.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- Paymasters: The `ergsPerPubdata: utils.DEFAULT_ERGS_PER_PUBDATA_LIMIT` in the transaction `customData` should be updated to `gasPerPubdata: utils.DEFAULT_GAS_PER_PUBDATA_LIMIT`. Custom paymasters are required to return a magic value after a transaction validation on the `validateAndPayForPaymasterTransaction` method.
+- If your project uses Account Abstraction, keep in mind that the IAccount interface has changed. We’ve introduced account versioning to allow for future updates. Accounts are required to return a magic value after a transaction validation on the `validateTransaction` method.
+- As with any other regenesis, it will remove all balances and contracts so you’ll need to deposit funds and redeploy your contracts again.
+
```
Comment (krofax) [inline docs/dev/troubleshooting/changelog.md:28]: Can you first state regenesis happened, before stating what they should do. @uF4No

**Review by krofax — CHANGES_REQUESTED** [review]

File: docs/dev/troubleshooting/changelog.md (hunk 4 above)
Comment (uF4No) [reply docs/dev/troubleshooting/changelog.md:28]: I already mention it in the first section, in bold 😉

File: docs/dev/troubleshooting/changelog.md (hunk 2 above)
Comment (uF4No) [reply docs/dev/troubleshooting/changelog.md:21]: confirmed with @Deniallugo that they'll released the docker images later today

File: docs/api/js/getting-started.md [hunk 5]
```diff
@@ -3,6 +3,12 @@
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+Please note that in the new `0.13.0` SDK the API layer operates with gas. The ergs concept is used by VM only.
+
+:::
```
Comment (AnastasiiaVashchuk) [inline docs/api/js/getting-started.md:10]: ::: warning

Please note, that in the new `0.13.0` SDK version, the API/Node layer operates with gas. The ergs concept is used by VM only.

:::

@uF4No  what we rephrase it like this?

Comment (krofax) [reply docs/api/js/getting-started.md:10]: Done.

**Pull request merged**